composer.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I understand the scripts but why do we need to write it.Maybe I don't understand the use of composer.json file

Reply:
composer.json describes the external dependency, what code depends on. 
The name and the author's part are not important at all. You can change it to your name and name it whatever you like 
The main goal of this file is the "require" section. There I added external dependency
`twig/twig`  you can see its repository location [here](https://github.com/twigphp/Twig), the "^3.0" says install any version of the external dependency with version 3.0 and above like (3.1, 3.2 3.4, etc.).

When you run 'composer install,' it will download the right version's code and put it in the vendor directory.

Reply:
got it